mungegithub/mungers/publish_scripts/clientgo_publish.sh

```diff
@@ -43,6 +44,23 @@ if git diff --cached --exit-code &>/dev/null; then
     exit 0
 fi
 git commit -m "${MESSAGE}"
+
+# Run "godep restore" to restore dependencies. Because entries for
```

Comment:
Prefix with "client-go's /vendor directory in staging doesn't include the other k8s.io/... dependencies, specifically apimachinery. Therefore, we do a restore/save cycle to fix this before publishing to the client repo.